### ISSUE TITLE
[Deque Analysis] Edit Item, Bitstreams Tab: "Content should be in a data table but is not" (Critical Issue)

### ISSUE THREAD

**Issue by tdonohue** [DSpace/dspace-angular#1270]
> Critical: This issue results in blocked content for people with disabilities, and will definitely prevent them from accessing fundamental features or content. This type of issue puts your organization at risk. Prioritize fixing as soon as possible, within the week if possible. Remediation should be a top priority. 

## Deque Analysis Summary

(Issue ID: 470946) Edit Item, Bitstreams tab: "Content should be in a data table but is not."  The page appears visually as a table, but it is not represented as a `<table>`.  Ideally, this will be fixed as follows:

Fix this issue by doing ALL of the following:
* Markup data that appears logically as a table using `<table>`, `<tr>`, and `<td>` elements.
* Ensure that header cell and data cell relationships are correctly conveyed according to the complexity of the table.
* If possible, add a `<caption>` element as the first child of the `<table>` element containing the name or title of the table.
* See also  https://dequeuniversity.com/class/semantic-structure2/tables/

The full issue (which includes links for more info) can be found at this link (requires login): https://axeauditor.dequecloud.com/test-run/0856438a-a19a-11eb-bc31-b7d5be387c86/issues?activeTab=dt-issue&page=0&pageSize=100&sortField=ordinal&sortDir=asc&row=9&filter%5Bseverity%5D=4&filter%5Bpage_number%5D=12&filter%5Bpage_number%5D=13&filter%5Bpage_number%5D=14&filter%5Bpage_number%5D=15

## More Information / Tools

* Consider using [Deque's free Chrome plugin](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) to check your work

**Comment by atarix83** [comment]
@tdonohue 

the solution of this issue is not trivial since it requires a complete refactoring of the components in order to be rendered as a table.
The estimation is about 60hours and in our opinion benefits are not commensurate with the cost

**Comment by tdonohue** [comment]
@atarix83 : Thanks for your honest estimate & opinion on this ticket.  Given that feedback, I'll add your estimate to this ticket...and I'll *decrease* the priority to `medium priority` for now.  I'll also flag it as needing a volunteer, in case there are others out there interested in such a refactor, even if it only provides accessibility benefits.

**Comment by artlowel** [comment]
Atmire would like to claim this issue as well as https://github.com/DSpace/dspace-angular/issues/3154 and https://github.com/DSpace/dspace-angular/issues/1191, and refactor this page to solve them all

**Comment by artlowel** [comment]
@tdonohue This already had an estimate of 60 hours. That looks correct
